cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/Main.java

```diff
@@ -245,7 +245,6 @@ public void execute() {
             }
 
             Path sourceRootPath = LauncherUtils.getSourceRootPath(sourceRoot);
-            System.setProperty("ballerina.source.root", sourceRootPath.toString());
```

Comment:
Could removing this break any functionality ?

Reply:
No this will not break any functionality and this system property is added only to give users the ability to retrieve the source root path via the Config API

Reply:
👍